docs/tutorial-code/redis.go

```diff
@@ -120,7 +120,7 @@ func main() {
 
 	if args.All || args.Metrics {
 		ms := integration.NewMetricSet("RedisSample")
-		fatalIfErr(populateMetrics(ms))
+		fatalIfErr(populateMetrics(&ms))
```

Comment:
If this has been changed, then also tutorial documentation should be. It appears there couple of times. https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/tutorial.md